### ISSUE TITLE
Audio file not saved to device

### ISSUE THREAD

**Issue by PaulNealeGit** [Hiraqui/cordova-plugin-ringtone#1]
I'm trying to get this plugin working with a simple test application, but the selected audio file isn't being stored on the Android device.  It's name is being added to the ringtone and notification list, but nothing happens when I try and play it.  I've had a look through the directory structure of the device and it doesn't appear to be saved anywhere.

**Comment by Hiraqui** [comment]
Hello Paul, this version of the plugin doen't export your file to another directory and sets it as ringtone. The plugin changes the same file properties and makes it avaiable on the ringtone screen. About the sound,I found out that some emulators don't play custom sounds, please try it on a device if you can. And let me know the file extension if you still have problems. I'm thinking of adding an export option on next version btw.

**Comment by PaulNealeGit** [comment]
Hi.  Thanks for the quick response.  Yes I thought that when I re-read the script, but I don't have much Java experience so wasn't sure.  I was trying on an actual device, a Sony Xperia E3 phone, using an mp3 file.  I got the 'success' callback in javascript, and the ringtone details were added to the list on my phone, but no sound was attached.  I tried with and without /android_asset/www/ but neither worked.
